python/heterocl/harness/stratus/utils.h

```diff
@@ -0,0 +1,268 @@
+#ifndef UTILS
+#define UTILS
+
+#include "hw_spec.h"
+#include <iostream>
+#include <fstream>
+
+std::string printBits(size_t const size, void const * const ptr) {
+    unsigned char *b = (unsigned char*) ptr;
+    bool byte;
+    int i, j;
```

Comment:
Please be consistent and strict with one coding style. For example, consistently initializing loop variable in the for statement.
```
    for (int i = size-1; i >= 0; i--) {
        for (int j = 7; j >= 0; j--) {
            byte = (b[i] >> j) & 1;
            ss << byte;
        }
    }
```
Also, please use `clang-format` to format all your C++ code.

Reply:
`utils.h` will be removed, it's a temporary file for the demo.

Reply:
Fixed by 3493876b0ab57e48e9f6a525e5d1c91922c1377f